config/initializers/apipie.rb

```diff
@@ -15,6 +15,7 @@
   config.default_locale = FastGettext.default_locale
   config.locale = ->(loc) { loc ? FastGettext.set_locale(loc) : FastGettext.locale }
 
+  ActiveRecord::Migration.maintain_test_schema!
```

Comment:
I made it load the schema at this point if there's no db yet, because calling Operatingsystem in the lines below will try to setup the model. Setting up the model involves calling `attributes_names`  on `validate_inclusion_in_families`, which breaks tests if there is no db, as it will not setup any validation (`attributes_names` is empty w/o a db)